### ISSUE TITLE
training loss of kontext and qwen edit lora is not quite normal

### ISSUE THREAD

**Issue by mliand** [kohya-ss/musubi-tuner#500]
The training loss of kontext and qwen edit lora is not quite normal. The normal loss should be within the range of 0.1, but the test loss is all within the range of 0.01.

**Comment by kohya-ss** [comment]
The absolute value of loss varies depending on the type of model, dataset, etc., so it is difficult to say what range is good. If the training results are good, there is no need to worry too much about the loss value.
